plugins/SegmentEditor/stylesheets/segmentation.less

```diff
@@ -366,10 +367,17 @@ html.comparisonsDisabled .segmentationContainer ul.submenu {
 }
 
 .segmentEditorPanel.expanded .add_new_segment {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
     width: 100%;
-    margin: 16px 0 8px 0;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+    margin: 1rem 0 0.5rem 0;
```

Comment:
Just a note, no need to change it again, but a shorter `margin: 1rem 0 0.5rem;` is identical here.